packages/core/contracts/oracle/interfaces/SlashingLibraryInterface.sol

```diff
@@ -46,13 +46,16 @@ interface SlashingLibraryInterface {
      * @param totalStaked The total amount of tokens staked.
      * @param totalVotes The total amount of votes.
      * @param totalCorrectVotes The total amount of correct votes.
+     * @param isGovernance Whether the request is a governance request.
+     * @param priceRequestId The price request ID.
      * @return wrongVoteSlashPerToken The amount of tokens to slash for voting wrong.
      * @return noVoteSlashPerToken The amount of tokens to slash for not voting.
      */
     function calcSlashing(
         uint256 totalStaked,
         uint256 totalVotes,
         uint256 totalCorrectVotes,
-        bool isGovernance
+        bool isGovernance,
+        bytes32 priceRequestId
```

Comment:
`bytes32 priceRequestId` is not used currently but gives us more flexibility for future slashing logic implementations